forge-gui/res/cardsfolder/upcoming/emergent woodwurm.txt

```diff
@@ -0,0 +1,13 @@
+Name:Emergent Woodwurm
+ManaCost:6 G
+Types:Creature Wurm
+PT:4/4
+K:Backup:3:BackupAbility
+SVar:BackupAbility:DB$ Animate | Triggers$ AttackTrig | sVars$ AE
+SVar:AttackTrig:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigDig | TriggerDescription$ Whenever this creature attacks, look at the top X cards of your library, where X is its power. You may put a permanent card with mana value X or less from among them onto the battlefield. Put the rest on the bottom of your library in a random order.
+SVar:TrigDig:DB$ Dig | DigNum$ TriggeredAttacker$CardPower | ChangeNum$ 1 | Optional$ True | ChangeValid$ Permanent.cmcLEX | DestinationZone$ Battlefield | DestinationZone2$ Library | LibraryPosition$ -1 | RestRandomOrder$ True
```

Comment:
Need to work with X here for ChangeValid

Reply:
ah right, I knew I forgot to test one of them 😩